### PR TITLE
Misc updates

### PR DESCRIPTION
**Pull request by MattEqualsCoder**
Removes some warnings, adds a few additional tracker lines, and fixes a bug I ran into with testing around hint tiles with dungeon requirements.

### CODE REVIEW

**Review by CPColin — APPROVED** [review]

File: src/Randomizer.Data/Configuration/Yaml/Sassy/hint_tiles.yml [hunk 1]
```diff
@@ -3,6 +3,10 @@
   - Text: 'I guess you''re too lazy to go there yourself? Fine. It says {0}'
   - Text: 'Can''t read it yourself? It says {0}'
   - Text: 'From what I''ve heard, it says {0}'
+  - Text: It says visit Shaktool.<break strength='1s'/>Fine, it says {0}.
```
Comment (Vivelin) [inline src/Randomizer.Data/Configuration/Yaml/Sassy/hint_tiles.yml:6]: Is this the right syntax? I thought strength was for the relative terms. I think duration is a different attribute? I could be wrong.

File: src/Randomizer.Data/Configuration/Yaml/Sassy/items.yml [hunk 2]
```diff
@@ -663,6 +663,10 @@
     "20":
     - Text: Okay, you have enough missiles now
     "21":
+    "40":
+    - Text: Someone must have a lot of time on their hands to get all of the missle packs
```
Comment (Vivelin) [inline src/Randomizer.Data/Configuration/Yaml/Sassy/items.yml:667]: Is "missles" here on purpose for pronunciation reasons?

**Review by Vivelin — APPROVED** [review]

File: src/Randomizer.Data/Configuration/Yaml/Sassy/hint_tiles.yml (hunk 1 above)
Comment (MattEqualsCoder) [reply src/Randomizer.Data/Configuration/Yaml/Sassy/hint_tiles.yml:6]: I'll double check in the morning my time. I think I just copied and pasted from elsewhere, but nothing to say where I pulled it from isn't broken.

File: src/Randomizer.Data/Configuration/Yaml/Sassy/items.yml (hunk 2 above)
Comment (MattEqualsCoder) [reply src/Randomizer.Data/Configuration/Yaml/Sassy/items.yml:667]: I believe so? I noticed the others were like that, so I followed suit. I'm guessing it was originally meant the avoid tracker saying it like miss-aisles. (Honestly we should have a line or two about that - "I hope you don't miss with these miss-aisles" and "tracked more missiles... Or is it miss-aisles? Whatever. Explosive arrows.")

**Review by Vivelin — APPROVED** [review]

**Pull request merged**